WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -1816,8 +1816,26 @@ public void CalculateDescriptor()
 
         if (ResolvedData is IRedArray ary)
         {
+            if (PropertyName == "compiledData" && ary is [IRedArray { Count: 3 } childAry])
```

Comment:
What is this for?

Reply:
factory.csv entries (the comment probably got lost when I refactored)